datalad/core/distributed/tests/test_clone.py

```diff
@@ -427,3 +427,57 @@ def test_installationpath_from_url():
     # and the hostname alone
     eq_(_get_installationpath_from_url("http://hostname"), 'hostname')
     eq_(_get_installationpath_from_url("http://hostname/"), 'hostname')
+
+
+# https://github.com/datalad/datalad/issues/3958
+@with_tempfile(mkdir=True)
+@with_tempfile(mkdir=True)
+def test_expanduser(srcpath, destpath):
+    src = Dataset(Path(srcpath) / 'src').create()
+    dest = Dataset(Path(destpath) / 'dest').create()
+
+    with chpwd(destpath), patch.dict('os.environ', {'HOME': srcpath}):
+        res = clone('~/src', 'dest', result_xfm=None, return_type='list',
```

Comment:
I guess it doesn't matter functionally (because this test [passes on windows][0]), but for consistency with the calls below should this be `op.join('~', 'src')`?

[0]: https://github.com/datalad/datalad/runs/374362978#step:8:25

Reply:
True, done.